### PR TITLE
Make SfpUtilHelper.get_physical_to_logical noexcept as in SfpUtilBase

### PR DESCRIPTION
**Pull request by vboykox**

### CODE REVIEW

File: sonic_platform_base/sonic_sfp/sfputilhelper.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def read_porttab_mappings(self, porttabfile):
     def get_physical_to_logical(self, port_num):
         """Returns list of logical ports for the given physical port"""
 
-        return self.physical_to_logical[port_num]
+        return self.physical_to_logical.get(port_num)
```
Comment (akokhan) [inline sonic_platform_base/sonic_sfp/sfputilhelper.py:126]: What's the benefit of this approach? But if we change here, we should change in other places as well..

Comment (vboykox) [reply sonic_platform_base/sonic_sfp/sfputilhelper.py:126]: Please take a look on these files:
+ https://github.com/Azure/sonic-platform-daemons/blob/master/sonic-xcvrd/scripts/xcvrd#L906
We expect for get_physical_to_logical to return None in the case of unknown port.
+ https://github.com/Azure/sonic-platform-common/blob/master/sonic_platform_base/sonic_sfp/sfputilbase.py#L603
That's SfpUtilBase implementation of the interface used in xcvrd 
No special benefit, just a fix.

Comment (vboykox) [reply sonic_platform_base/sonic_sfp/sfputilhelper.py:126]: I guess by other places you mean get_logical_to_physical - no reason for change if everything conforms the usage)

Comment (akokhan) [reply sonic_platform_base/sonic_sfp/sfputilhelper.py:126]: Good. Please go ahead and create PR in community with this explanation.

**Pull request closed without merging**